src/test/ui/span/method-and-field-eager-resolution.stderr

```diff
@@ -1,22 +1,26 @@
 error[E0282]: type annotations needed
-  --> $DIR/method-and-field-eager-resolution.rs:5:5
+  --> $DIR/method-and-field-eager-resolution.rs:4:9
    |
 LL |     let mut x = Default::default();
-   |         ----- consider giving `x` a type
-LL |     x.0;
-   |     ^ cannot infer type
+   |         ^^^^^
    |
    = note: type must be known at this point
+help: consider giving `x` an explicit type
+   |
+LL |     let mut x: _ = Default::default();
+   |              +++
```

Comment:
Note to self: rework the output so we special case `: _` with a different placeholder.